code/datums/ai/_ai_controller.dm

```diff
@@ -379,10 +379,7 @@ multiple modular subtrees with behaviors
 	switch(ai_status)
 		if(AI_STATUS_ON)
 			START_PROCESSING(SSai_behaviors, src)
-		if(AI_STATUS_OFF)
-			STOP_PROCESSING(SSai_behaviors, src)
-			CancelActions()
-		if(AI_STATUS_IDLE)
+		if(AI_STATUS_OFF, AI_STATUS_IDLE)
 			STOP_PROCESSING(SSai_behaviors, src)
 			CancelActions()
```

Comment:
I found these two branches were identical while researching this so I just merged them.